pkg/doc/verifiable/credential_test.go

```diff
@@ -355,6 +355,34 @@ func TestValidateVerCredIssuer(t *testing.T) {
 		require.Error(t, err)
 		require.Contains(t, err.Error(), "issuer: Invalid type")
 	})
+
+	t.Run("test verifiable credential with object context", func(t *testing.T) {
```

Comment:
probably "with string issuer"

Reply:
Missed to update the name. Thanks. I have updated it [now](https://github.com/hyperledger/aries-framework-go/compare/915c66c0ad4904b420459fef1797603726e67474..e7c610cb5ba36606a28670ce46fb69dc36bdcc10).